dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/ConfigSpringBootApplicationClassloaderTests.java

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.springboot.support.fat;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.config.ClassloaderElement;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.websphere.simplicity.config.SpringBootApplication;
+
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.utils.HttpUtils;
+
+@RunWith(FATRunner.class)
+public class ConfigSpringBootApplicationClassloaderTests extends AbstractSpringTests {
+    @Override
+    public Set<String> getFeatures() {
+        return new HashSet<>(Arrays.asList("springBoot-1.5", "servlet-3.1"));
+    }
+
+    @Override
+    public String getApplication() {
+        return SPRING_BOOT_15_APP_BASE;
+    }
+
+    @Override
+    public AppConfigType getApplicationConfigType() {
+        return AppConfigType.SPRING_BOOT_APP_TAG;
+    }
+
+    // DO NOT OVERRIDE expectAppSuccess()
+
+    @Test
+    public void testSpringBootApplicationClassloaderDefaults() throws Exception {
+        server.stopServer(false);
```

Comment:
I think we should have a more general way to do this that does not require a server restart in the test execution.  Create a new method in AbstractSpringTests:

```
public void modifyConfiguration(ServerConfiguration config) {
    // do nothing by default
}
```
Then this test class can override the method and add the classloader element to the SpringBootApplication already contained in the ServerConfiguration.
This method would get called by `AbstractSpringTests.configureServer()` just before doing the call to `server.updateServerConfiguration(config)`